src/WebJobs.Extensions.DurableTask/LinuxAppServiceFileLogger.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+#if !FUNCTIONS_V1
+using Mono.Unix.Native;
+#endif
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// TBD>.
+    /// </summary>
+    public class LinuxAppServiceFileLogger
+    {
+        private readonly string logFileName;
+        private readonly string logFileDirectory;
+        private readonly string logFilePath;
+        private readonly string archiveFilePath;
+        private readonly BlockingCollection<string> buffer;
+        private readonly List<string> currentBatch;
+        private readonly CancellationTokenSource cancellationTokenSource;
+        private Task outputTask;
+
+        /// <summary>
+        /// TBD.
```

Comment:
Do you plan on filling in these TBDs or should be just remove them?

Reply:
@cgillum I do plan on filling them. I just pushed that commit / did the port late last night so I didn't want to necessarily comment at that point, especially because  I did not know what the performance results would be. I'll get to it shortly